test/spec.js

```diff
@@ -1,9 +1,231 @@
 // spec.js
-describe('Protractor Demo App', function() {
-  it('should have a title', function() {
+describe('Clonage App', function() {
 
-    browser.get('/');
+	beforeEach(function() {
+		browser.get('/');
+		browser.waitForAngular();
+	});
+
+
+	it('should have a title', function() {
+		expect(browser.getTitle()).toEqual('CLONAGE');
+	});
+
+	it('should direct to the signup page', function() {
+		expect(browser.getCurrentUrl()).toMatch(/\//);
+	});
+
+	describe('When registering as a user', function() {
+
+		var clonageSignup;
+
+		beforeEach(function() {
+			clonageSignup = new ClonageSignupPage();
+			// clonageSignup.get();
+		});
+
+		// afterEach(function() {
```

Comment:
Ideally, you don't want to commit commented out code.

It can make the file a bit hard to read and possibly lead to confusion.